packaging/installer/netdata-uninstaller.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-#shellcheck disable=SC2181
+#shellcheck disable=SC2181,2059,SC2155
```

Comment:
Did you mean SC2059 ?

```suggestion
#shellcheck disable=SC2181,SC2059,SC2155
```